snuba/datasets/configuration/generate_config_docs.py

```diff
@@ -0,0 +1,31 @@
+from tempfile import TemporaryDirectory
+
+import jsonschema2md
+
+from snuba import settings
+from snuba.datasets.configuration.json_schema import V1_ALL_SCHEMAS
+
+CONFIGURATION_DOCS_PATH = f"{settings.ROOT_REPO_PATH}/docs/source/configuration"
+
+
+if __name__ == "__main__":
+    for component, schema in V1_ALL_SCHEMAS.items():
+        with TemporaryDirectory() as temp_dir:
+            temp_filename = f"{temp_dir}/{component}"
+            parser = jsonschema2md.Parser(
+                examples_as_yaml=False,
+                show_examples="all",
+            )
+            md_lines = parser.parse_schema(schema)
+            reformatted_md_lines = []
+            for line in md_lines:
+                # jsonschema2md adds some extra lines of its own. This removes them.
+                if "**Items**" not in line:
+                    line = line.replace("`", "")
+                    line = line.replace(" Cannot contain additional properties.", "")
+                    reformatted_md_lines.append(line)
```

Comment:
Can you share what the bad `jsonschema2md` output looks like? Just having trouble picturing it.

Reply:
Yeah, so there are two things which `jsonschema2md` adds. Firstly, whenever there is a key with `TYPE_STRING_ARRAY`, there will be an extra `Items` bullet point without any type. See `columns` and `query_processors` keys. The description on the the upper level key is already sufficient. Secondly, if any `properties` contain the `additionalProperties: False` key/value, then the "Cannot contain any additional properties" string is added. This gets very repetitive and causes some confusion.  

![image](https://user-images.githubusercontent.com/23560545/192372859-cf2136b5-1820-4cef-883c-e33b57a1839b.png)